frontend/src/pages/org/archived-item-detail/archived-item-detail.ts

```diff
@@ -683,6 +683,11 @@ export class ArchivedItemDetail extends TailwindElement {
           ? html`<div id="replay-crawl" class="aspect-4/3 overflow-hidden">
               <replay-web-page
                 source="${replaySource}"
+                url="${String(
+                  this.crawl.seedCount === 1
+                    ? ifDefined(this.crawl.firstSeed)
+                    : "",
+                )}"
```

Comment:
This might create an edge case of converting `nothing` into a string, so we can remove `ifDefined`, and simplify as:
```suggestion
                url="${this.crawl.seedCount === 1 && this.crawl.firstSeed || ""}"
```